demo/admin/src/products/future/generated/ManufacturersGrid.tsx

```diff
@@ -0,0 +1,255 @@
+    GQLManufacturersGridQueryVariables,
+} from "./ManufacturersGrid.generated";
+
+const manufacturersFragment = gql`
+    fragment ManufacturersGridFuture on Manufacturer {
+        id
+        address {
+            street
+            streetNumber
+            alternativeAddress {
+                street
+                streetNumber
+            }
+        }
+        addressAsEmbeddable {
+            street
+            streetNumber
+            alternativeAddress {
+                street
+                streetNumber
+            }
+        }
+    }
+`;
+
+const manufacturersQuery = gql`
+    query ManufacturersGrid($offset: Int, $limit: Int, $sort: [ManufacturerSort!], $search: String, $filter: ManufacturerFilter) {
+        manufacturers(offset: $offset, limit: $limit, sort: $sort, search: $search, filter: $filter) {
+            nodes {
+                ...ManufacturersGridFuture
+            }
+            totalCount
+        }
+    }
+    ${manufacturersFragment}
+`;
+
+const deleteManufacturerMutation = gql`
+    mutation DeleteManufacturer($id: ID!) {
+        deleteManufacturer(id: $id)
+    }
+`;
+
+const createManufacturerMutation = gql`
+    mutation CreateManufacturer($input: ManufacturerInput!) {
+        createManufacturer(input: $input) {
+            id
+        }
+    }
+`;
+
+function ManufacturersGridToolbar() {
+    return (
+        <Toolbar>
+            <ToolbarAutomaticTitleItem />
+            <ToolbarItem>
+                <GridToolbarQuickFilter />
+            </ToolbarItem>
+            <ToolbarItem>
+                <GridFilterButton />
+            </ToolbarItem>
+            <ToolbarFillSpace />
+            <ToolbarActions>
+                <Button startIcon={<AddIcon />} component={StackLink} pageName="add" payload="add" variant="contained" color="primary">
+                    <FormattedMessage id="manufacturer.newManufacturer" defaultMessage="New Manufacturer" />
+                </Button>
+            </ToolbarActions>
+        </Toolbar>
+    );
+}
+
+export function ManufacturersGrid(): React.ReactElement {
+    const client = useApolloClient();
+    const intl = useIntl();
+    const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("ManufacturersGrid") };
+
+    const columns: GridColDef<GQLManufacturersGridFutureFragment>[] = [
+        {
+            field: "id",
+            headerName: intl.formatMessage({ id: "manufacturer.id", defaultMessage: "ID" }),
+            filterable: false,
+            sortable: false,
+            flex: 1,
+            minWidth: 150,
+        },
+        {
+            field: "address_street",
+            headerName: intl.formatMessage({ id: "manufacturer.address.street", defaultMessage: "Street" }),
+            filterable: false,
+            sortable: false,
+            valueGetter: ({ row }) => row.address?.street,
+            flex: 1,
+            minWidth: 150,
+        },
+        {
+            field: "address_streetNumber",
```

Comment:
Why don't you use dot as the separator (as in the config)?

Reply:
because mui-datagrid uses the field-name for filtering. and the generated api does not (and probably can't) use dot-notation for filter-fields.
<img width="534" alt="image" src="https://github.com/vivid-planet/comet/assets/1324250/acd1684e-2dfc-47c0-b9e6-7570fef0cd27">